apps/condo/domains/organization/integrations/sbbol/sync/syncSubscriptions.js

```diff
@@ -89,9 +89,9 @@ const syncSubscriptionsFor = async (advanceAcceptance) => {
                 ...advanceAcceptance, // TODO: Figure out, why it crashes here on `payerInn` field
             },
         })
-        debugMessage('Created trial subscription for SBBOL', trialServiceSubscription)
+        logger.info({ message: 'Created trial subscription for SBBOL', trialServiceSubscription })
     } else {
-        debugMessage(`User from organization(inn=${payerInn}) has declined our offer in SBBOL`)
+        logger.info({ message: `User from organization(inn=${payerInn}) has declined our offer in SBBOL`, payerInn })
```

Comment:
it's better to remove context from the message (inn)

Reply:
Is it private data?

Reply:
Removed